source/Octopus.Tentacle/Security/AesEncryption.cs

```diff
@@ -60,7 +60,7 @@ Aes GetCryptoProvider(byte[]? iv = null)
             var provider = Aes.Create();
             provider.Mode = CipherMode.CBC;
             provider.Padding = PaddingMode.PKCS7;
-            provider.KeySize = 128;
+            provider.KeySize = 256;
```

Comment:
I was expecting you'd need to modify the key generation to use a 32 byte key also.

Reply:
setting this key size doesn't do anything by itself, when you set the key, it'll then set the keysize 😂 

Maybe steal the Calamari version: https://github.com/OctopusDeploy/Calamari/pull/1405/files#diff-67547aea4be6e349d7da9a9a3f90d83a795171716b038c406aff1ff8d4c8d18b